src/main/kotlin/systems/ajax/malov/stockanalyzer/controller/StockRecordsController.kt

```diff
@@ -13,15 +15,14 @@ import systems.ajax.malov.stockanalyzer.service.StockRecordAnalyzerService
 class StockRecordsController(private val stockRecordAnalyzerService: StockRecordAnalyzerService) {
 
     @GetMapping("/bestFive")
-    fun getFiveBestStockSymbolsWithStockRecords(): ResponseEntity<AggregatedStockRecordResponseDto> = ResponseEntity
-        .ok(
-            toAggregatedStockItemResponseDto(
-                stockRecordAnalyzerService
-                    .getFiveBestStockSymbolsWithStockRecords()
-            )
-        )
+    fun getFiveBestStockSymbolsWithStockRecords(): Mono<AggregatedStockRecordResponseDto> =
+        stockRecordAnalyzerService
+            .getFiveBestStockSymbolsWithStockRecords()
+            .map {
+                toAggregatedStockItemResponseDto(it)
+            }
 
-    @GetMapping("/symbols")
-    fun getAllManageableStockSymbols(): ResponseEntity<List<String>> =
-        ResponseEntity.ok(stockRecordAnalyzerService.getAllManageableStocksSymbols())
+    @GetMapping("/symbols", produces = [MediaType.TEXT_EVENT_STREAM_VALUE])
+    fun getAllManageableStockSymbols(): Flux<String> =
```

Comment:
I don't think it should be stream. IMO stream means that requested data may be changed in the future and updates will be delivered in stream. Here you have a finite amount of data that will be delivered nearly instantly and the stream will be closed. So for me, it looks more like Mono<List<String>>